### PR TITLE
Improve flask logging message

### PR DESCRIPTION
**Pull request by shifucun**
* Remove DEBUG message!
* Let mixer error message show up..

### CODE REVIEW

File: server/services/datacommons.py [hunk 1]
```diff
@@ -435,8 +434,8 @@ def send_request(req_url,
     if response.status_code != 200:
         raise ValueError(
             'Response error: An HTTP {} code was returned by the mixer. '
-            'Printing response\n{}'.format(response.status_code,
-                                           response.reason))
+            'Printing response:\n{}'.format(response.status_code,
```
Comment (beets) [inline server/services/datacommons.py:437]: should we keep reason as well?

**Review by beets — APPROVED** [review]
nice update

File: server/services/datacommons.py (hunk 1 above)
Comment (shifucun) [reply server/services/datacommons.py:437]: Added back

**Comment by shifucun** [comment]
Thanks!

**Pull request merged**